extensions/amp-access/amp-access.md

```diff
@@ -482,26 +482,6 @@ This section will cover a detailed explanation of the design underlying the amp-
 [12]: #metering
 [13]: #first-click-free
 
-## Validation errors
+## Validation
 
-The following lists validation errors specific to the `amp-access` tag
-(see also `amp-access` in the [AMP validator specification](https://github.com/ampproject/amphtml/blob/master/extensions/amp-access/0.1/validator-amp-access.protoascii)):
```

Comment:
Final language: See [<amp extension> rules](https://github.com/ampproject/amphtml/blob/master/extensions/<amp-extension>/0.1/validator-<amp-extension>.protoascii) in the AMP validator specification.